.github/workflows/e2e-staging.yml

```
@@ -6,7 +6,7 @@ on:
   pull_request:
     branches: [ main, develop ]
   issue_comment:
-    types: [created]
+    types: [ created ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)